doc/ref_arch/kubernetes/chapters/appendix-a.md

```diff
@@ -1,18 +1,21 @@
 [<< Back](../../kubernetes)
-# Appendix A - Guidance for VNF-to-CNF Transition
+# Appendix A - Guidance For VNF Vendors
```

Comment:
Should this be "Guidance for CNF Vendors"; RA-1 has a  "Guidance for VNF Vendors"

Reply:
@pgoyal01 It includes both because we also have a section for `Guidance for VNF-to-CNF Transition`

Reply:
@deebhatia I saw that but I was referring to the Appendix Title -- currently it refer only to VNF; we have spent a lot of energy changing from VNF and NFV to include explicit references to CNF.

Reply:
I've changed the title to "Appendix A - Guidance For application Vendors"